web/templates/settings/admin_linkifier_list.hbs

```diff
@@ -1,7 +1,7 @@
 {{#with linkifier}}
-<tr class="linkifier_row{{#if ../can_modify}} movable-row{{/if}}" data-linkifier-id="{{id}}">
+<tr class="linkifier_row{{#if (and ../can_modify ../can_drag)}} movable-row{{/if}}" data-linkifier-id="{{id}}">
     <td>
-        {{#if ../can_modify}}
+        {{#if (and ../can_modify ../can_drag)}}
```

Comment:
I think this part belongs in the second commit, right?

Reply:
Oops, fixed!